focuspoints.lrdevplugin/focus_points/pentax/pentax k-50.txt

```diff
@@ -2,8 +2,6 @@
 -- Submitted by: roguephysicist
 -- 16.3 MP, 4928x3264
 -- AF System: 11 points, 9 cross-type
--- selected points: AFPointSelected
--- in-focus points: AFPointsInFocus
```

Comment:
[I did it for my python program](https://github.com/roguephysicist/PyFocus/blob/master/PentaxK50.py), and agree that there's no easy way to enable this for every camera. I even tried drawing in the viewfinder lines and such, but that was a serious pain in the ass with little benefit.

Ultimately, I think it's already pretty great to have an accurate representation of the focus points. She shape and size of the points are mostly an aesthetic thing.